Code/EnginePlugins/JoltPlugin/System/JoltWorldModule.cpp

```diff
@@ -797,6 +797,14 @@ void ezJoltWorldModule::UpdateSettingsCfg()
 void ezJoltWorldModule::ApplySettingsCfg()
 {
   SetGravity(m_Settings.m_vObjectGravity, m_Settings.m_vCharacterGravity);
+
+  if (m_pSystem)
+  {
+    auto physicsSettings = m_pSystem->GetPhysicsSettings();
+    physicsSettings.mPointVelocitySleepThreshold = m_Settings.m_fSleepVelocityThreshold;
```

Comment:
Are there other thresholds?

Reply:
There are tons of settings. But none super relevant, at least not for now.